tensorflow/core/util/sparse/sparse_tensor.h

```diff
@@ -148,6 +148,13 @@ class SparseTensor {
                                          const int split_dim,
                                          const int num_split);
 
+  // Slice() will slice the input SparseTensor into a SparseTensor based on
+  // specified start and size.
+  template <typename T>
+  static SparseTensor Slice(const SparseTensor& tensor,
+                            const gtl::ArraySlice<int64> &start,
```

Comment:
TF's code guidelines require the amperstand to be part of the type:  const gtl::ArraySlice<int64>& start